tests/runtime/basic

```diff
@@ -220,11 +220,17 @@ PROG BEGIN { @ = 1; @a[1] = 1; delete(@); delete(@a[1]); printf("ok\n"); exit();
 EXPECT ok
 TIMEOUT 1
 
+NAME delete multiple-map
+PROG BEGIN { @ = 1; @a[1] = 1; delete(@, @a[1]); printf("ok\n"); exit(); }
+EXPECT ok
```

Comment:
Without the deletion, this would also print the map right?

Reply:
Yes I believe that's correct.

Reply:
The problem with the `EXPECT ok` approach is that the test will still pass even if the `delete` call is omitted (or doesn't work correctly).

The existing tests in this file (and in much of the rest of the test suite) are crappy - no fault of your own, @arthurshau! It's a big problem we're going to have to deal with gradually.

I think to properly check that our code is doing what it's meant to, we'd have to do something like:
```
EXPECT_NONE @: 1
EXPECT_NONE @a[1]: 1
```

(don't bother fixing all the old tests right now, it'd just be good to have new tests be better examples to copy from in the future)